articles/value-part.html

```diff
@@ -123,7 +123,7 @@ <h3>Two Kinds of Pointer Types in Go</h3>
 </p>
 
 <p>
-We have learned <a href="pointer.html">Go pointers</a> in the last last article.
+We have learned <a href="pointer.html">Go pointers</a> in the last article.

```

Comment:
Ah, here it should be "the article before the last".
I used many "last last" and "next next" before, and I have corrected most of them, but missed this one. :)

Reply:
thanks for pointing this out :) I fixed it as requested